src/box/sql/select.c

```diff
@@ -2131,11 +2131,11 @@ sqlColumnsFromExprList(Parse * parse, ExprList * expr_list,
 			}
 			zName = sqlMPrintf("%.*z_%u", nName, zName, ++cnt);
 		}
+		assert(zName != NULL);
```

Comment:
The `while` loop above checks `zName` of `NULL` in the loop condition, which implies that `zName` can in fact be `NULL` here, so this won't silence static analyzers.

Reply:
Squashed this patch.